app.yml

```diff
@@ -42,6 +42,8 @@ scale:
 environment:
   COUNTRY_CODE: ca
   ALEMBIC_DATABASE_URI: postgresql+psycopg2://$(SQL_TRACKER_STORE_USER):$(SQL_TRACKER_STORE_PASSWORD)@$(SQL_TRACKER_STORE_URL):5432/$(SQL_TRACKER_STORE_DB)
+  DAILY_CHECKIN_BASE_URL: "https://covid19.dialogue.co/#/rasa/ci"
```

Comment:
Won't the URL change between environments?

Reply:
Nop